### ISSUE TITLE
Basic Navigation not working in Safari desktop browser

### ISSUE THREAD

**Issue by gedw99** [PolymerElements/polymer-starter-kit#451]
Environment:
OSX
PSK 1.1.1 (no changes)

Browser:
Chrome Version 46.0.2490.71 (64-bit)
Safari Version 9.0 (11601.1.56)

Server:
Gulp Serve (as standard in PSK)

Steps to reproduce:
1. Download PSK 1.1.1
2. gulp serve
3. test in both browser clicking the top nav or left draw nav. Both have the same issue

Basically, on Safari, for all menu Navigation on the side drawer requires pressing twice.

Demo video showing Chrome and Safari side by side:
http://youtu.be/GGXHfnm_iYk

I tried with with and without hashbang setting in the router.html, and it makes no difference.

**Comment by Nejat** [comment]
I've experienced the same behavior. 
It only happens if you click the `text label` or `icon`. 
Click outside of either, in the white space, and it works with one click.
**Perhaps the event is not bubbling up correctly?**

\* *Additionally, `Firefox` exhibits the same behavior, though `Opera` works as expected.*

**Comment by chuckh** [comment]
Looks like this is a duplicate issue see #166.  @gedw99 should we close this?

**Comment by gedw99** [comment]
@chuckh 
YES looks the same.  Thanks for addressing this.

To check, i just tried PSK (1.1.1) again on the following:
- FF (41.0.2) on OSX
- Safari (9.0 (11601.1.56)) on OSX
- Edge on Windows ( VirtualBox)

FF & Safari have the exact same problem.
Edge does not have this problem.

The issue is:
- when click on the text link, does not work.
- when click on area (non text) link, works.

My markup for the let Nav and the Top nav is using:

````

<paper-menu class="list" attr-for-selected="data-route" selected="[[route]]">

      <a data-route="home" href="/">
        <span>Home</span>
      </a>

      <a data-route="product" href="/products">
        <span>Products</span>
      </a>

....

```


Also i tried , Edge on Windows (VirtualBox) on: https://elements.polymer-project.org/browse?package=paper-elements

Lots of little bugs.

**Comment by chuckh** [comment]
Fix menu not working in FF 38 #470 fixed this issue.
